src/main/scala/com/codacy/client/bitbucket/v2/service/BuildStatusServices.scala

```diff
@@ -62,4 +67,11 @@ class BuildStatusServices(client: BitbucketClient) {
 
     client.putJson(Request(url, classOf[BuildStatus]), payload)
   }
+
+  private def generateCommitBuildStatusesUrl(owner: String, repository: String, commit: String): String = {
+    val encodedOwner = URLEncoder.encode(owner, "UTF-8")
```

Comment:
Can'y you encode just one time, in the end?

Reply:
What do you mean?